### PR TITLE
Sam colab version control

### PR DESCRIPTION
**Pull request by Nanayeb34**
# Description

Added a summary doc on how to integrate Google Colab with github for version control purposes

# Reviewers

@mentions of 1 person at minimum to review the proposed changes
@jojoboateng @wsedor

### CODE REVIEW

File: docs/COLAB_VERSION_CONTROL.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+**Colab Version Control**
+
+In line with best practices, developers using Google colaboratory to write their scripts are encouraged to integrate their notebooks into Github for version control.
+
+**Steps to integrate Colab with Github**
+1. Clone the nsmqai repositiory and create a local branch. All branches must have the right naming conventions. See [here](https://github.com/nsmq-ai/nsmqai/blob/34bca778b61e3cca6532805c37eae9e74d18a8a5/docs/CONTRIBUTING.md) for more information on how to name your branch.
+2. Push the local branch with no new changes to Github in order to create a remote branch. 
+3. Open your notebook in Google Colaboratory.
+4. Click on 'File' and select 'Save a copy in Github'.
+5. A pop up window will appear and you will be asked to authenticate your Google account with your Github accoount if you haven't already done so.
```
Comment (wsedor) [inline docs/COLAB_VERSION_CONTROL.md:10]: typo in Github `acoount`

Comment (Nanayeb34) [reply docs/COLAB_VERSION_CONTROL.md]: error has been rectified

File: docs/COLAB_VERSION_CONTROL.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+**Colab Version Control**
+
+In line with best practices, developers using Google colaboratory to write their scripts are encouraged to integrate their notebooks into Github for version control.
+
+**Steps to integrate Colab with Github**
+1. Clone the nsmqai repositiory and create a local branch. All branches must have the right naming conventions. See [here](https://github.com/nsmq-ai/nsmqai/blob/34bca778b61e3cca6532805c37eae9e74d18a8a5/docs/CONTRIBUTING.md) for more information on how to name your branch.
```
Comment (wsedor) [inline docs/COLAB_VERSION_CONTROL.md:6]: minor typo in `repositiory`

**Review by wsedor — APPROVED** [review]
looks good to me. left a comment on a typo. please address it and go ahead with the merge

**Pull request merged**